src/BugsnagUnity/Client.cs

```diff
@@ -95,6 +95,31 @@ public Client(INativeClient nativeClient)
             {
                 // Async behavior is not available in a test environment
             }
+            AddBugsnagLoadedBreadcrumb();
+        }
+
+        private void AddBugsnagLoadedBreadcrumb()
+        {
+            if (IsUsingFallback() && Configuration.IsBreadcrumbTypeEnabled(BreadcrumbType.State))
+            {
+                Breadcrumbs.Leave("Bugsnag loaded", BreadcrumbType.State, null);
+            }
+        }
+
+        private bool IsUsingFallback()
+        {
+            switch (Application.platform)
+            {
+                case RuntimePlatform.OSXEditor:
+                case RuntimePlatform.OSXPlayer:
```

Comment:
Wouldn't `OSXPlayer` leave a duplicate? The `bugsnag-cocoa` integration should be passing this in automatically.

Reply:
ah yes good point